sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/README.md

```diff
@@ -0,0 +1,123 @@
+# Azure Functions Bindings for Azure SignalR Service
```

Comment:
Can you take a look at the format used in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/README.md and match it?